src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/integrasjon/ecb/ECBIntegrationTest.kt

```diff
@@ -0,0 +1,109 @@
+package no.nav.familie.ks.sak.no.nav.familie.ks.sak.integrasjon.ecb
+
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.verify
+import no.nav.familie.ks.sak.OppslagSpringRunnerTest
+import no.nav.familie.ks.sak.config.DatabaseCleanupService
+import no.nav.familie.ks.sak.integrasjon.ecb.ECBService
+import no.nav.familie.ks.sak.integrasjon.ecb.domene.ECBValutakursCacheRepository
+import no.nav.familie.valutakurs.Frequency
+import no.nav.familie.valutakurs.ValutakursRestClient
+import no.nav.familie.valutakurs.domene.ECBExchangeRate
+import no.nav.familie.valutakurs.domene.ECBExchangeRateDate
+import no.nav.familie.valutakurs.domene.ECBExchangeRateKey
+import no.nav.familie.valutakurs.domene.ECBExchangeRateValue
+import no.nav.familie.valutakurs.domene.ECBExchangeRatesData
+import no.nav.familie.valutakurs.domene.ECBExchangeRatesDataSet
+import no.nav.familie.valutakurs.domene.ECBExchangeRatesForCurrency
+import no.nav.familie.valutakurs.domene.toExchangeRates
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import java.math.BigDecimal
+import java.time.LocalDate
+
+@ExtendWith(MockKExtension::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ECBIntegrationTest : OppslagSpringRunnerTest() {
+    @MockK
+    private lateinit var ecbClient: ValutakursRestClient
+
+    @Autowired
+    private lateinit var ecbService: ECBService
+
+    @Autowired
+    private lateinit var ecbValutakursCacheRepository: ECBValutakursCacheRepository
+
+    @Autowired
+    private lateinit var databaseCleanupService: DatabaseCleanupService
+
+    @BeforeEach
+    fun setUp() {
+        ecbService =
+            ECBService(
+                ecbClient = ecbClient,
+                ecbValutakursCacheRepository = ecbValutakursCacheRepository,
+            )
+        databaseCleanupService.truncate()
+    }
+
```

Comment:
```suggestion
    @SpykBean
    private lateinit var ecbClient: ValutakursRestClient

    @Autowired
    private lateinit var ecbService: ECBService

    @Autowired
    private lateinit var ecbValutakursCacheRepository: ECBValutakursCacheRepository
```


Noen små forandringer som kan vurderes:
1. Vi trenger ikke truncate her. Siden denne klassen arver fra OppslagSpringRunnerTest, så gjøres det automagisk for oss (Se AfterEach'n i OppslagSpringRunnerTest)
2. Istedenfor å kjøre en beforeeach fordi vi vil bruke en mocked ecbClient, så kan vi bare bruke SpykBean annotasjonen. 

Gjør vi disse to forandringene så kan vi fjerne beforeeach.